src/OrchardCore.Cms.Web/appsettings.json

```
@@ -113,6 +113,9 @@
     //      "RequestUrlPrefix": "tenant"
     //    }
     //  ]
+    //},
+    //"OrchardCore_HealthChecks": {
+    //  "Url": "/health"
```

I prefer to show the default in the samples:

```suggestion
    //  "Url": "/health/live"
```